utils/templates/README-module+member.md.in

```diff
@@ -114,15 +114,15 @@ Variables
 
 Variable | Description | Required
 -------- | ----------- | --------
-`ipaadmin_principal` | The admin principal is a string and defaults to `admin` | no
```

Comment:
This is more a question than a suggestion: should we use "yes/no" or "true/false" to say that a parameter is required or not?

This is not an Ansible/YAML boolean, it is documentation, so I lean for "yes/no", but I'm really not sure.

Reply:
I have been thinking about this also and wanted to discuss this.

Reply:
So, I prefer yes/no, in this case.